cfme/tests/infrastructure/test_individual_host_creds.py

```diff
@@ -38,14 +37,20 @@ def get_host_data_by_name(provider_key, host_name):
     return None
 
 
+def get_credentials_from_config(credential_config_name):
+    creds = conf.credentials[credential_config_name]
+    username = ("{}\\{}".format(creds.domain, creds.username) if creds.get('domain') else
```

Comment:
@psav @jkrocil i see this domain trick of config credentials multiple time, i believe we should add it as a generally supported item in Base Credentials as a followup

else we just start to litter the codebase with credentials helpers again

Reply:
actually we do - its jsut strangely hidden

https://github.com/ManageIQ/integration_tests/blob/master/cfme/base/credential.py#L82

@quarckster as far as i can tell we handle that `\` magic in the credential base, please remove own implementations and defer to the real implementation